Core/clim-basic/windowing/pixmap.lisp

```diff
@@ -36,6 +36,9 @@
 (defgeneric medium-copy-area (from-drawable from-x from-y width height
                               to-drawable to-x to-y))
 
+(defmethod pixmap-mirror ((pixmap pixmap))
```

Comment:
Since backends use this, shouldn't there be a `defgeneric` and shouldn't it be exported from `clime` or `climb`?

Reply:
it is just a temporary prothesis before the next merge request (in bm-2 branch) that removes the interface (that is to make pull requests smaller, I could do it at one go), so I don't want to commit to it by advertising it as an exported symbol.

re defgeneric, it is already in this file, belowe defclass pixmap.

Reply:
in other words this interface was already part of the core.